pkg/importer/http-datasource.go

```diff
@@ -131,13 +131,13 @@ func (hs *HTTPDataSource) Info() (ProcessingPhase, error) {
 		return ProcessingPhaseTransferDataDir, nil
 	}
 	if hs.readers.Convert {
-		if hs.brokenForQemuImg || hs.readers.Archived || hs.customCA != "" {
-			return ProcessingPhaseTransferScratch, nil
-		}
-	} else {
-		if hs.readers.Archived || hs.customCA != "" {
-			return ProcessingPhaseTransferDataFile, nil
-		}
+		// removing check for hs.brokenForQemuImg, and always assuming it is true
+		// revert once we are able to get nbdkit 1.35.8, which contains a fix for the
+		// slow download speed.
+		return ProcessingPhaseTransferScratch, nil
+	}
+	if hs.readers.Archived || hs.customCA != "" {
+		return ProcessingPhaseTransferDataFile, nil
 	}
 	hs.url, _ = url.Parse(fmt.Sprintf("nbd+unix:///?socket=%s", nbdkitSocket))
```

Comment:
We need to keep it around for the multi stage import, which uses nbdkit. But you are right this will still be slow for uncompressed raw files since it uses nbdkit.